test/boost/expr_test.cc

```diff
@@ -7,6 +7,7 @@
 #include "types/types.hh"
 #include "types/list.hh"
 #include "types/map.hh"
+#include "types/vector.hh"
```

Comment:
> boost/expr_test: add vector expression tests
> 
> I've added/adjusted tests using vector_constructor and collection_constructor with list_or_vector style before preparation.

Use imperative mood, please

> 
> Implemented utilities used in expr_test similar to those added in https://github.com/zpp-2024-vector-search/scylladb/commit/8f6309bd660084f802cae7746a5f9ed3bcaa4685

Reply:
The commit message has been changed accordingly.